.github/workflows/continuous-integration-workflow.yml

```diff
@@ -36,7 +36,12 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-go@v2
         with:
-          go-version: 1.17
+          go-version: 1.18
+      - name: windows - downgrade mingw to 10.2.0
+        # TODO revert once 'go test -race' issue is fixed
```

Comment:
should we make a followup ticket for this?

Reply:
I wondered about that, but seems like we'd be checking on it each grooming to determine if it could be worked yet...

Reply:
Yeah, but I think that's a separate issue. Biggest concern is just having this fall through the cracks, if it's something we'd like to check on in a month or two.

Reply:
Maybe we can preemptively schedule it for 3-4 sprints from now or something?

Reply:
I created a ticket but wasn't able to assign it to a future sprint.